guacamole/src/main/webapp/translations/en.json

```diff
@@ -2,6 +2,8 @@
     
     "NAME" : "English",
     
+    "DIRECTION": "ltr",
```

Comment:
Please follow the style of the surrounding code. The JSON here includes a space on either side if the `:`, as should any new code.